### DISCUSSION TITLE
Fill element with text

### DISCUSSION THREAD

**Discussion by AErmolchev** [QuestPDF/QuestPDF#492] [Q&A] [answered]
Hello! First of all thanks a lot for this product. And here's my question: is there any way to make text fill all of element's area (I have rectangle and know its top left corner, heigth and width) without defining font size? Or may be therre's some way to calculate it correctly?

**Comment by DawidRobertNowak** [comment, marked as answer]
Hello,

We are closing this thread due to inactivity. We apologize for the delayed response. Thank you for the kind words and for taking the time to reach out.

The `ScaleToFit` element automatically scales content down to fit the available space, which should address your use case without needing to manually calculate font size. You can find the documentation here:
https://www.questpdf.com/api-reference/scale.html

Should you encounter any further problems, please don't hesitate to open a new issue.

Thank you!
